org.openhab.binding.zigbee.ember/src/main/java/org/openhab/binding/zigbee/ember/handler/EmberHandler.java

```diff
@@ -43,6 +48,22 @@
 public class EmberHandler extends ZigBeeCoordinatorHandler implements FirmwareUpdateHandler {
     private final Logger logger = LoggerFactory.getLogger(EmberHandler.class);
 
+    private ScheduledFuture<?> pollingJob = null;
```

Comment:
Very minor comment, just in case you want to change: No need to assign `null` as this is the default anyways; I, personally, prefer to omit assigning `null` to fields as it always looks a little odd at first read (probably personal preference).